packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.scss

```diff
@@ -5,16 +5,24 @@
 }
 
 .svg {
-  width: 20px;
-  height: 20px;
-  padding: 4px;
+  width: 32px;
+  height: 32px;
+  padding: 3px;
   box-sizing: content-box;
 }
 
-:global(.ms-Fabric.is-focusVisible) .cell {
+:global(.ms-Fabric.is-focusVisible) .cell, :global(.ms-Fabric.is-focusVisible) .circleCell {
```

Comment:
This can be replaced or used with the focus mxin below as well. O tjhoml at least.

Reply:
I have no idea! haven't done any focus stuff in fabric yet